src/apps/fuzzers/fuzzerResolutions.c

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 Uber Technologies, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/** @file
+ * @brief Fuzzer program for resolution specific functions
+ */
+
+#include "aflHarness.h"
+#include "h3api.h"
+#include "utility.h"
+
+typedef struct {
+    int res;
+} inputArgs;
+
+int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
+    if (size < sizeof(inputArgs)) {
+        return 0;
+    }
+    const inputArgs *args = (const inputArgs *)data;
+
+    // TODO: Do not perform bound checks
```

Comment:
Does this mean these functions are currently unsafe?

Reply:
Yes, this is addressed in #550. Once that is merged these can be fuzzed. As it is they crash almost instantly (consider resolution = 17)